datadog_checks_dev/datadog_checks/dev/tooling/manifest_validator/v2/validator.py

```diff
@@ -0,0 +1,82 @@
+#  (C) Datadog, Inc. 2020-present
+#  All rights reserved
+#  Licensed under a 3-clause BSD style license (see LICENSE)
+import json
+
+import requests
+
+import datadog_checks.dev.tooling.manifest_validator.common.validator as common
+from datadog_checks.dev.tooling.manifest_validator.common.validator import BaseManifestValidator
+
+METRIC_TO_CHECK_EXCLUDE_LIST = {
+    'openstack.controller',  # "Artificial" metric, shouldn't be listed in metadata file.
+    'riakcs.bucket_list_pool.workers',  # RiakCS 2.1 metric, but metadata.csv lists RiakCS 2.0 metrics only.
+}
+
+
+class SchemaValidator(BaseManifestValidator):
+    def validate(self, check_name, decoded, fix):
+        if not self.should_validate():
+            return
+
+        # Get API and APP keys which are needed to call Datadog API
+        org_name = self.ctx.obj['org']
+        if not org_name:
+            self.fail('No `org` has been set')
+            return
+
+        if org_name not in self.ctx.obj['orgs']:
+            self.fail(f'Selected org {org_name} is not in `orgs`')
+            return
+
+        org = self.ctx.obj['orgs'][org_name]
+
+        api_key = org.get('api_key')
+        if not api_key:
+            self.fail(f'No `api_key` has been set for org `{org_name}`')
+            return
```

Comment:
self.fail + returning will make this validation error out right? (not success with an error message?)

Reply:
Yes, removed the redundant `return` lines.